### PR TITLE
Delete button functional

### PR DESCRIPTION
**Pull request by youngj3**

### CODE REVIEW

File: client/src/components/PizzaPlaceShow.js [hunk 1]
```diff
@@ -46,7 +67,10 @@ const PizzaPlaceShow = props => {
 		newReview = <NewReviewForm 
 			setPizzaPlace={setPizzaPlace}
 			pizzaPlace={pizzaPlace}
+			deleteReview={deleteReview}
 		/>
+
+
```
Comment (nwalberts) [inline client/src/components/PizzaPlaceShow.js]: delete extra lines

File: client/src/components/ReviewItem.js [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
-const ReviewListItem = ({ title, rating, text }) => {
+const ReviewListItem = ({ title, rating, text, deleteReview, id, currentUser }) => {
+
+  const handleDeleteClick = event => {
+		event.preventDefault()
+		deleteReview(id)
+	  }
+
+    let button = ""
+    if (currentUser) {
```
Comment (nwalberts) [inline client/src/components/ReviewItem.js]: this button should only display for an admin user or for the user that created the review. Can we update to check for this?

File: server/src/routes/api/v1/pizzaPlaceReviewRouter.js [hunk 3]
```diff
@@ -22,4 +22,12 @@ pizzaPlaceReviewRouter.post("/", async (req, res) => {
...
 })
 
+pizzaPlaceReviewRouter.delete("/:id", async (req, res) => {
+  try {
+    await Review.query().deleteById(req.params.id)
+    return res.status(204).json({message: 'deletion success'})
+  } catch(error) {
+		return res.status(500).json({errors: error})
+	}
+})
```
Comment (nwalberts) [inline server/src/routes/api/v1/pizzaPlaceReviewRouter.js:32]: it could just be my view, but is the indentation off here?

**Review by nwalberts — CHANGES_REQUESTED** [review]

File: server/src/routes/api/v1/pizzaPlaceReviewRouter.js (hunk 3 above)
Comment (youngj3) [reply server/src/routes/api/v1/pizzaPlaceReviewRouter.js:32]: i promise it doesn't look like that on my machine

**Pull request merged**